### DISCUSSION TITLE
How to harmonise target SNPs and proxy SNPs?

### DISCUSSION THREAD

**Discussion by zhuziwei5225** [MRCIEU/TwoSampleMR#258] [Q&A]
Hello,
I found proxy SNPs with vcftools. How can I correct the beta when the alleles of target SNP and proxy SNP do not correspond for the same SNP?  harmonise_data() will remove the ambigous SNP. 
For example:
exposure effect of target SNP =0.5
effect allele = A
other allele = T

outcome effect of proxy SNP =-0.05
effect allele = A
other allele = C
 
or

exposure effect of target SNP =0.5
effect allele = A
other allele = G

outcome effect of proxy SNP =-0.05
effect allele = A
other allele = C

Thank you.

**Comment by wangyiyisheng** [comment]
Have you addressed this problem?
